packages/opentelemetry-resource-util/src/index.ts

```diff
@@ -110,6 +116,16 @@ const MAPPINGS = {
     },
     [AWS_ACCOUNT]: {otelKeys: [SemanticResourceAttributes.CLOUD_ACCOUNT_ID]},
   },
+  [CLOUD_RUN_REVISION]: {
+    [LOCATION]: {otelKeys: [SemanticResourceAttributes.CLOUD_REGION]},
+    [SERVICE_NAME]: {otelKeys: [SemanticResourceAttributes.FAAS_NAME]},
+    [CONFIGURATION_NAME]: {otelKeys: [SemanticResourceAttributes.FAAS_NAME]},
+    [REVISION_NAME]: {otelKeys: [SemanticResourceAttributes.FAAS_VERSION]},
+  },
+  [CLOUD_FUNCTION]: {
+    [REGION]: {otelKeys: [SemanticResourceAttributes.CLOUD_REGION]},
+    [FUNCTION_NAME]: {otelKeys: [SemanticResourceAttributes.FAAS_NAME]},
```

Comment:
These mappings are used for both trace `g.co/r/` labels and monitoring monitored resource. These monitored resources are not currently writable for monitoring, so this presents an issue (we have the same issue in the java impl I believe https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/blob/60cf3d675eda0900258853d568c21bfd4d692d49/shared/resourcemapping/src/main/java/com/google/cloud/opentelemetry/resource/ResourceTranslator.java#L97

Should we go ahead with this or maybe I can add a `skipUnsupportedMonitoredResources` bool option?

Reply:
You can see what I did in go: https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/main/exporter/metric/metric.go#L343

Either option is fine with me

Reply:
Disabled auto-merge. I will update it so it works correctly on cloud run

Reply:
Updated L187 ptal

Reply:
appologies for waking a dead thread: could you possible explain "These monitored resources are not currently writable for monitoring" and/or point to reference documentation?